calculator.c

```diff
@@ -0,0 +1,117 @@
+#include<stdio.h>
+#include<string.h>
+#include<ctype.h>
+#include<stdlib.h>
+//store number in stack
+int numStack[30];
+int topnum=-1;
+//store operators in stack
+char opStack[20];
+int topop=-1;
+//Perform arithmetic operation
+void calculate(){
+    int x,y,z,n;
+    char op;
+    //To check there are minimum two operands
```

Comment:
Avoid usage of comments. Usage of proper variable and function name will lead code to self expressive. And comments can be avoided.